### PR TITLE
Emulating ora API if logLevel is verbose

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
First part of fixing https://github.com/blackbaud/skyux2/issues/2268

### CODE REVIEW

File: src/logger.js [hunk 1]
```diff
@@ -34,11 +34,24 @@ const logger = new winston.Logger({
...
 
+// Expose ora logger
+logger.promise = (message) => {
+
+  if (logger.logLevel !== 'verbose') {
+    return ora(message).start();
+  }
+
+  // Expose a compatabily API if verbose.
+  // This is necessary otherwise ora would hijack displaying it's message on a single line.
```
Comment (Blackbaud-PaulCrowder) [inline src/logger.js]: This should be "its" and not "it's"

Comment (Blackbaud-BobbyEarl) [reply src/logger.js]: Good catch.  Fixed.

File: src/logger.js [hunk 2]
```diff
@@ -34,11 +34,25 @@ const logger = new winston.Logger({
...
 logger.logLevel = logLevel;
 
+// Expose ora logger
+logger.promise = (message) => {
+
+  if (logger.logLevel !== 'verbose') {
+    return ora(message).start();
+  }
+
+  // Expose a compatabily API if verbose.
```
Comment (Blackbaud-SteveBrush) [inline src/logger.js]: "compatabily"

**Comment by Blackbaud-BobbyEarl** [comment]
Hey guys, slightly changed the functionality to better emulate the promise logging.

**Pull request merged**